bsconfig.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@float-dao/config",
+  "name": "@float-capital/config",
```

Comment:
Is this still desired?

Reply:
No, thanks for spotting. Changing to `float-packages`